setup.py

```diff
@@ -61,6 +61,7 @@ def run(self):
     "dulwich>=0.19.11",
     "ruamel.yaml>=0.15.91",
     "psutil==5.6.2",
+    "funcy>=1.12",
```

Comment:
there is compact implementation in collections.py. Let's remove it if we use funcy in other places now

Reply:
Ok